Source/UI/Scripts/GamingGroup/gamingGroup.js

```diff
@@ -33,6 +33,27 @@ Views.GamingGroup.GamingGroupView.prototype = {
         });
 
         this._googleAnalytics.trackGAEvent("GamingGroups", "GamingGroupRenamed", "GamingGroupRenamed");
+    },
+    renderNemeStatsPointsLineGraph: function () {
+        var url = '/api/v2/PlayedGames/?gamingGroupId=1';
+
+        $.get(url, function (data) {
```

Comment:
we prefer to use the $.ajax function because it allows passing of the query parameters as a setting and not concatenating them to the url